utils/documentation_tests.txt

```diff
@@ -54,3 +54,4 @@ src/transformers/models/wav2vec2/modeling_wav2vec2.py
 src/transformers/models/wav2vec2/tokenization_wav2vec2.py
 src/transformers/models/wav2vec2_with_lm/processing_wav2vec2_with_lm.py
 src/transformers/models/wavlm/modeling_wavlm.py
+src/transformers/models/ctrl/modeling_ctrl.py
```

Comment:
Small nit, but as this change is not related to DebertaV2Tokenizer, I think it would be better for the repo history to do it in another PR: do you mind opening another PR to isolate this change? :slightly_smiling_face:

Reply:
Yes sure, I have removed unrelated changes - but can I ask would it be possible that I open the PR after this has been merged. Will it be cleaner that way?

Reply:
Yes! Absolutely :slightly_smiling_face:

Reply:
I've opened another PR with these changes in:

https://github.com/huggingface/transformers/pull/16850#issue-1209499773

Could I trouble you to take a look to see if it seems fine?